.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+# see https://github.com/viamrobotics/build-action for help
+on:
+  push:
+    tags:
+    - '[0-9]+.[0-9]+.[0-9]+'
+    - '[0-9]+.[0-9]+.[0-9]+-rc'
+    - '[0-9]+.[0-9]+.[0-9]+-rc[0-9]+'
+    
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: viamrobotics/build-action@v1
+      with:
+        # note: you can replace this line with 'version: ""' if you want to test the build process without deploying
+        version: ${{ github.ref_name }}
+        ref: ${{ github.sha }}
+        key-id: ${{ secrets.viam_key_id }}
+        key-value: ${{ secrets.viam_key_value }}
+        
```

Comment:
I don't know what you did here, but Github still thinks this file doesn't end in a newline, even though it ends with a blank line! :laughing: You can tell because Github has labeled it with a big red :no_entry: , and you can tell locally because running `git diff` or similar should have a warning that the file doesn't have a newline. but as you say, this is under time pressure so we'll skip it for now.